runtime.md

```diff
@@ -33,24 +33,79 @@ This is provided so that consumers can find the container's configuration and ro
 }
 ```
 
-## Lifecycle
+## Operations
+
+OCI compliant runtimes MUST support the following operations, unless support for the operation can not be supported by the base operating system (kernel) itself.
+
+OPEN: we should consider specifying the options/args for each operation.
+For example, checkpoint (for runc) seems to have a lot of params.
 
 ### Create
 
-Creates the container: file system, namespaces, cgroups, capabilities.
+Using the `config.json` and `runtime.json` files, along with the file system path referenced in the `config.json`, this operation MUST create a new container.
+This includes creating the relevant namespaces, cgroups and configuring the appropriate capabilities for the container.
+
+### Start
+
+This operation MUST start a previously created container.
+Starting a container MUST create a new process, as specified by the `config.json` file within the scope of the container.
+Attempting to start an already running container MUST have no effect on the container and MUST generate an error.
+Starting a stopped container, one that had been previously be running, MUST create a new process, as specified by the `config.json` file, within the scope of the container.
+
+OPEN: the entire "generate an error" stuff in here is something we should discuss.
+My initial thought was that it would be better to generate an error so that the call would know exactly what happened - meaning, did something new take place or was the container already running. If we didn't return an error (or something) then the caller wouldn't know.  And they can always ignore errors if they want/need to.
+
+### Stop
+
+This operation MUST stop a running container.
+Stopping a container MUST stop all of the processes running within the scope of the container.
+Stopping a container MUST NOT delete the associated namespaces or cgroups for the container.
```

Comment:
Its kinda a hard question because it goes into how you define a container.  Technically I think its a bad idea to keep namespaces around because its buggy and racy just to make it fit someones made up definition of a container.  We would be jumping through hoops in code and on the system for no real use case other than making the technical reality match something that one of us typed. 

We dont keep around all the memory of a VM when its stopped just because we made a UI with a stop and delete button so I dont think it makes sense to do this.

Just because we want a stop and delete phase does not mean we need to go looking for something that we can keep laying around to clean up later just to support the thought.  Maybe we don't need a stop and delete at all so maybe we should discuss some reasons why people think it should be added.